src/main/java/seedu/address/logic/commands/DeleteRoleCommand.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.company.Company;
+import seedu.address.model.role.Role;
+import seedu.address.model.role.RoleManager;
+
+
+public class DeleteRoleCommand extends Command {
+    public static final String COMMAND_WORD = "deleteRole";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the role identified by the index number used in the displayed company list \n"
+            + "followed by the index number used by the role in the specified company.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1 1";
+
+    public static final String MESSAGE_DELETE_ROLE_SUCCESS = "Deleted Role: %1$s  %1$s";
```

Comment:
Not sure if you intended to have double spaces between the indices for aesthetic purposes but just letting you know!

Reply:
Did not notice the double spacing. Fixed. Thanks.